scripts/lit.js

```diff
@@ -6,20 +6,39 @@
 // Configuration:
 //
 // Commands:
-// randbot lit <message_link> - Immortalizes a message in #lit... fam
+//   hubot lit <message_link> - Immortalizes a message in #lit... fam
+//   hubot lit - Immortalizes a shared message in #lit... fam
 //
 // Author:
 //   mikestephens/harasho
 
+const responses = require('../responses.json');
```

Comment:
Try something here, I don't know if it's going to work but it might. Instead of 
```js
const responses = require('../responses.json');
```
do
```js
const { lit } = reqiure('../responses.json');
```

This way, we can just say `lit.success` instead of `responses.lit.success`.

Reply:
Yeah, totally going to work. I had my doubts as to use of `lit` as a top level variable.